packages/vscode/package.json

```diff
@@ -44,7 +44,6 @@
     "vscode": "^1.67.0"
   },
   "activationEvents": [
-    "onLanguage:astro",
```

Comment:
Any reason for this? Not all Astro workspaces have a Astro config file. (Ex: anonymous workspaces, fixtures etc)

Reply:
Ah. I was getting a lint warning that this was no longer needed in the latest versions, but probably wise to keep it for backwards compat.

Reply:
Ah yeah, I forgot about that change. They changed it in 1.74.0, but we support up to 1.67.0, so your assumption is correct that we should keep it!